zh/monitoring-by-tidb-monitor.md

```diff
@@ -0,0 +1,175 @@
+---
```

Comment:
Please also update the name of this file to make it consistent with TOC

Reply:
updated.